data/publiccloud/log_instance.sh

```diff
@@ -18,7 +18,7 @@ ec2_start_log()
     aws ec2 get-console-output --instance-id "$INSTANCE_ID" --output text > "${OUTPUT_DIR}/${CNT}_get_console_output_start.log"
     set -e
     nohup ssh $SSH_OPTS "ec2-user@${HOST}" -- sudo dmesg -c -w > "${OUTPUT_DIR}/${CNT}_dmesg.log" 2>&1 &
-    echo "$!" >> "${OUTPUT_DIR}/pid"
+    echo $! >> "${OUTPUT_DIR}/pid"
 }
```

Comment:
Using `""` in echo is always safer than without it, I'd remove this.

Reply:
Seems like `PID_FILE=${OUTPUT_DIR}/pid` fits here as well

Reply:
> Seems like `PID_FILE=${OUTPUT_DIR}/pid` fits here as well

nice catch ! 

> and I am not that sure about appending the PID to the file. Simple redirection seems to me cleaner.

I assume initial idea was that this file maybe called several times , but from I see in current implementation this was never the case so we can simplify this .